src/test/ui/traits/multidispatch-convert-ambig-dest.stderr

```diff
@@ -2,13 +2,18 @@ error[E0282]: type annotations needed
   --> $DIR/multidispatch-convert-ambig-dest.rs:26:5
    |
 LL |     test(22, std::default::Default::default());
-   |     ^^^^ cannot infer type for type parameter `U` declared on the function `test`
+   |     ^^^^ cannot infer type of the type parameter `U` declared on the function `test`
+   |
+help: consider specifying the generic arguments
+   |
+LL |     test::<T, U>(22, std::default::Default::default());
+   |         ++++++++
```

Comment:
I'm noticing that we already know `U` in the main label, we could use that to replace the other args with `_` in the suggestion.